lib/config/presets/index.ts

```diff
@@ -320,6 +321,10 @@ export async function resolveConfigPresets(
   let config: AllConfig = {};
   // First, merge all the preset configs from left to right
   if (inputConfig.extends?.length) {
+    // Compile templates
+    inputConfig.extends = inputConfig.extends.map((tmpl) =>
+      template.compile(tmpl, {}),
```

Comment:
you're passing an empty object so every handlebar template is replaced with an empty string. I don't know how your test can succeed 🤔

Reply:
`template.compile` uses
```typescript
const data = { ...GlobalConfig.get(), ...input, env };
```
where `input` is the method argument (`{}` in this case). I'm not sure where it makes sense to put custom data in that use case, since most of the time the input comes from `env`.